generators/client/templates/angular/src/main/webapp/app/login/login.component.ts.ejs

```diff
@@ -33,18 +33,13 @@ export class LoginComponent implements OnInit, AfterViewInit {
 
   authenticationError = false;
 
-  loginForm = this.fb.group({
-    username: [null, [Validators.required]],
-    password: [null, [Validators.required]],
-    rememberMe: [false],
+  loginForm = new FormGroup({
+    username: new FormControl('', { nonNullable: true, validators: [Validators.required] }),
```

Comment:
isn't it risky?

Reply:
Replacing null value par en empty string doesn't mean the same thing: 
 Here, for example the required validator is useless...

Reply:
In this case `''` is used for the type, `''` and `null` are treated equally.
Example if we set `null`, the user types something and deletes, we have `''` or `null`? So they are the same.
`''` is used just for the type (string).